packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,63 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+function range(file, node, source, color, CONTEXT = 2) {
```

Comment:
`CONTEXT` is the number of lines before and after the node to display. I put it in a parameter just to avoid having a magic number inside the function, but I could also make it a constant if you think that would be better.